tools/buildsteps/windows/vswhere.bat

```diff
@@ -57,7 +59,7 @@ IF %vcvars%==no (
     IF NOT ERRORLEVEL 1 (
       IF EXIST "%%i\VC\vcvarsall.bat" (
         SET vcvars="%%i\VC\vcvarsall.bat"
-        SET vsver=14
+        SET vsver=14 2015
```

Comment:
Not sure how exact the cmake "-A" works with the VS, does it still work then on 2015? 

Know e.g. there that also no ARM64 is present (where works by binary addons on 2017).

Reply:
I think it should work but I haven't tested it (I don't have VS 2015). 

According CMake doc. is supported: 
https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2014%202015.html